PassiFy/PasswordGenerator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace PassiFy
+{
+    public class PasswordGenerator
+    {
+        private readonly Random _random = new Random();
+        public string GeneratePassword(int length)
```

Comment:
Each method/function should only have one purpose. You can google Single Responsibility Principle for further info. This method just generates password, the other just selects characters. This helps make testing easier as well.